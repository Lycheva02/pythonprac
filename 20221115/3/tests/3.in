import time

start1 = time.time()
alp = Alpha()
for i in range(1000000):
	alp.a = i
	alp.b = i
	alp.c = i
	alp.d = i
	alp.e = i
	alp.f = i
	alp.g = i
	alp.h = i
	alp.i = i
	alp.j = i
	alp.k = i
	alp.l = i
	alp.m = i
	alp.n = i
	alp.o = i
	alp.p = i
	alp.q = i
	alp.r = i
	alp.s = i
	alp.t = i
	alp.w = i
	alp.x = i
	alp.y = i
	alp.z = i

a = str(alp)
stop1 = time.time()
print(a)

start2 = time.time()
alq = AlphaQ()
for i in range(1000000):
	alq.a = i
	alq.b = i
	alq.c = i
	alq.d = i
	alq.e = i
	alq.f = i
	alq.g = i
	alq.h = i
	alq.i = i
	alq.j = i
	alq.k = i
	alq.l = i
	alq.m = i
	alq.n = i
	alq.o = i
	alq.p = i
	alq.q = i
	alq.r = i
	alq.s = i
	alq.t = i
	alq.w = i
	alq.x = i
	alq.y = i
	alq.z = i

a = str(alq)
stop2 = time.time()
print(a)
assert((stop1-start1) < (stop2-start2))