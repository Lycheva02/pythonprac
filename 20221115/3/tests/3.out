a: 999999, b: 999999, c: 999999, d: 999999, e: 999999, f: 999999, g: 999999, h: 999999, i: 999999, j: 999999, k: 999999, l: 999999, m: 999999, n: 999999, o: 999999, p: 999999, q: 999999, r: 999999, s: 999999, t: 999999, w: 999999, x: 999999, y: 999999, z: 999999
a: 999999, b: 999999, c: 999999, d: 999999, e: 999999, f: 999999, g: 999999, h: 999999, i: 999999, j: 999999, k: 999999, l: 999999, m: 999999, n: 999999, o: 999999, p: 999999, q: 999999, r: 999999, s: 999999, t: 999999, w: 999999, x: 999999, y: 999999, z: 999999
